tests/plugins/tabletools/manual/colordialogposition.html

```diff
@@ -0,0 +1,21 @@
+<div id="editor">
+	<table border="1" cellpadding="1" cellspacing="1" style="width:500px">
+		<tbody>
+			<tr>
+				<td>1.1</td>
+				<td>1.2</td>
+			</tr>
+			<tr>
+				<td>2.1</td>
+				<td>2.2</td>
+			</tr>
+		</tbody>
+	</table>
+</div>
+
+<script>
+	if ( bender.tools.env.mobile ) {
```

Comment:
Would be nice to also add manual test for mobiles, as the regression which has been introduced here is caused by changes dedicated to mobiles. I think that if we change test steps with dev console, it should be possible to do it in the same test file, WDYT?

Reply:
Yup, should work 👍